integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
+  },
+  "devDependencies": {
+    "@percy-io/react-percy": "^0.1.5",
+    "webpack": "^3.5.5"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "percy": "react-percy"
```

Comment:
Also, to your point about naming, it would totally make sense if this said `"percy": "percy-snapshot"`